integration-tests/features/component_analysis_v2_smoke_tests.feature

```diff
@@ -0,0 +1,121 @@
+Feature: Smoke Tests Component Analysis v2
+
+   Scenario Outline: Check the component analysis REST API endpoint for 20 most popular NPM components
+    Given System is running
+    Given Three scale preview service is running
+      
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+      When I start v2 component analyses <ecosystem>/<package>/<version> with user_key
+      Then I should get 200 status code
+        And I should receive a valid JSON response
+
+      When I look at the component analysis duration
+      Then I should see that the component analysis duration is less than 10 second
```

Comment:
i dont think it complies to our slo. please check the timings.

Reply:
according to SLO the LSP call happens within 3 seconds but pavel said it can also be somewhat late and if it increases more than 10 thats not good

Reply:
so 10 seconds is for combined results of 20 pkgs? or for individual pkg? In either case, i dont think its compatible with our slo

Reply:
reduced the call to 5 secs because i was getting 3.66 secs for each call made it 5 to be on safer side